### ISSUE TITLE
Normalized function in Pinocchio Quaternion leads to segfaul under -march=native flag 

### ISSUE THREAD

**Issue by cmastalli** [stack-of-tasks/eigenpy#268]
The following Python code leads to segfaul if you compile `eigenpy` and `pinocchio` with `-march=native` flag.
```python
import pinocchio
import numpy as np

R = np.random.rand(3, 3)

# This leads to segtaul
print(pinocchio.Quaternion(R).normalized().coeffs())

# Instead, this patches the issue
quat = pinocchio.Quaternion(R)
quat.normalize() # Note that this is not normalized()
print(quat.coeffs())
```

fyi @wxmerkt

**Comment by jcarpent** [comment]
@cmastalli march native is hardware dependent. Please profile the compilation flags associated with the current context.

**Comment by jcarpent** [comment]
@cmastalli Could you create a docker image to replicate the current behavior?

**Comment by jcarpent** [comment]
Just to let you know everything goes well on my side.

**Comment by cmastalli** [comment]
I compile the code in Release mode using clang-10. And I just included -march=native.

Furthermore, I installed eigenpy by source in the same way that Pinocchio. Boost, eigen are installed by binaries in a Ubuntu 20.04 OS and Python 3.8

Let me know if there something else that you would like to know

**Comment by jcarpent** [comment]
Solved by #269
